gensrc/script/doris_builtins_functions.py

```diff
@@ -533,6 +533,10 @@
             '15FunctionContextERKNS1_9StringValERKNS1_6IntValES6_'],
     [['length'], 'INT', ['VARCHAR'],
             '_ZN5doris15StringFunctions6lengthEPN9doris_udf15FunctionContextERKNS1_9StringValE'],
+    [['char_length'], 'INT', ['VARCHAR'],
+            '_ZN5doris15StringFunctions16char_utf8_lengthEPN9doris_udf15FunctionContextERKNS1_9StringValE'],
+    [['character_length'], 'INT', ['VARCHAR'],
+            '_ZN5doris15StringFunctions16char_utf8_lengthEPN9doris_udf15FunctionContextERKNS1_9StringValE'],
```

Comment:
This can be merged in one line.
```suggestion
    [['char_length', 'character_length'], 'INT', ['VARCHAR'],
            '_ZN5doris15StringFunctions16char_utf8_lengthEPN9doris_udf15FunctionContextERKNS1_9StringValE'],
```

Reply:
done @imay